website/content/v1.1/talos-guides/configuration/editing-machine-configuration.md

```diff
@@ -28,6 +28,9 @@ Each of these commands can operate in one of four modes:
 > Note: applying change on next reboot (`--mode=staged`) doesn't modify current node configuration, so next call to
 > `talosctl edit machineconfig --mode=staged` will not see changes
 
+Additionally, there is also `talosctl get machineconfig`, which retrieves the current node configuration API resource and contains the machine configuration in the `.spec` field.
```

Comment:
I'm torn on this one. If you try to extract the `.spec` field with `yq`, it does slight reformatting, so what you get is not byte-by-byte what you submitted.

On other hand we have `talosctl read /system/state/config.yaml` which does exactly what you submitted, but this one looks weird. 

It's more of a question I'm struggling with myself, rather than related to the PR :)

Reply:
I struggled with this one too, and that may be an indicator that we need to figure out a better way of doing this.

In the context of the original conversation and page topic, I don't think a byte-for-byte copy is really warranted or needed, especially if the user has gone through interactive setup as they haven't "submitted" anything themselves.

Reply:
or we could have a new `talosctl machineconfig` similar to `talosctl kubeconfig` which just wraps the reading of `/system/state/config,yaml`